src/OrchardCore.Modules/OrchardCore.Workflows/Views/WorkflowTrimming.Fields.Edit.cshtml

```diff
@@ -0,0 +1,111 @@
+            <label class="form-check-label" for="@Html.IdFor(m => m.Statuses[1])">
+                @T["Starting"]
+            </label>
+        </div>
+    </div>
+
+    <div class="col-xs-12">
+        <div class="form-check">
+            <input type="hidden" asp-for="@Model.Statuses[2].Status" />
+            <input class="form-check-input workflow-status" type="checkbox" asp-for="@Model.Statuses[2].IsSelected" id="@Html.IdFor(m => m.Statuses[2])">
+            <label class="form-check-label" for="@Html.IdFor(m => m.Statuses[2])">
+                @T["Resuming"]
+            </label>
+        </div>
+    </div>
+
+    <div class="col-xs-12">
+        <div class="form-check">
+            <input type="hidden" asp-for="@Model.Statuses[3].Status" />
+            <input class="form-check-input workflow-status" type="checkbox" asp-for="@Model.Statuses[3].IsSelected" id="@Html.IdFor(m => m.Statuses[3])">
+            <label class="form-check-label" for="@Html.IdFor(m => m.Statuses[3])">
+                @T["Executing"]
+            </label>
+        </div>
+    </div>
+
+    <div class="col-xs-12">
+        <div class="form-check">
+            <input type="hidden" asp-for="@Model.Statuses[4].Status" />
+            <input class="form-check-input workflow-status" type="checkbox" asp-for="@Model.Statuses[4].IsSelected" id="@Html.IdFor(m => m.Statuses[4])">
+            <label class="form-check-label" for="@Html.IdFor(m => m.Statuses[4])">
+                @T["Halted"]
+            </label>
+        </div>
+    </div>
+
+    <div class="col-xs-12">
+        <div class="form-check">
+            <input type="hidden" asp-for="@Model.Statuses[5].Status" />
+            <input class="form-check-input workflow-status" type="checkbox" asp-for="@Model.Statuses[5].IsSelected" id="@Html.IdFor(m => m.Statuses[5])">
+            <label class="form-check-label" for="@Html.IdFor(m => m.Statuses[5])">
+                @T["Finished"]
+            </label>
+        </div>
+    </div>
+
+    <div class="col-xs-12">
+        <div class="form-check">
+            <input type="hidden" asp-for="@Model.Statuses[6].Status" />
+            <input class="form-check-input workflow-status" type="checkbox" asp-for="@Model.Statuses[6].IsSelected" id="@Html.IdFor(m => m.Statuses[6])">
+            <label class="form-check-label" for="@Html.IdFor(m => m.Statuses[6])">
+                @T["Faulted"]
+            </label>
+        </div>
+    </div>
+
+    <div class="col-xs-12">
+        <div class="form-check">
+            <input type="hidden" asp-for="@Model.Statuses[7].Status" />
+            <input class="form-check-input workflow-status" type="checkbox" asp-for="@Model.Statuses[7].IsSelected" id="@Html.IdFor(m => m.Statuses[7])">
+            <label class="form-check-label" for="@Html.IdFor(m => m.Statuses[7])">
+                @T["Aborted"]
+            </label>
+        </div>
+    </div>
```

Comment:
Why you didn't iterate over the workflow statuses instead?!

Reply:
See the comment above:

> @* We use a hard-coded list of statuses here instead of iterating over the enum so the explicit T-strings make
   localization easier (and so the default English displayed string can be different than the enum value's name). *@

Reply:
While the values are English I don't think there's an issue

Reply:
What's not an issue, you mean?

Reply:
Localization issue

Reply:
It's about extracting the PO strings. I've clarified it in the comment.